### DISCUSSION TITLE
Sign-In with Delegation shows page not found

### DISCUSSION THREAD

**Discussion by AkshadaPramodKadam** [Azure/api-management-developer-portal#2638] [General]
## Bug description

When we do sign-in with Delegation signin-sso shows page not found
The delegation settings enable with sign-in and product subscription.
https://contoso.developer.azure-api.net/signin-sso?token=&returnUrl=%2Freturn%2Furl

## Reproduction steps

1.Sign in via delegated login
2.When redirected to the developer portal /signin-sso page you receive the "Page you're looking for doesn't exist" message.

## Expected behavior

With Delegation user should be sign-in successfully and redirect to home page

## Is your portal managed or self-hosted?

Self-hosted

## Release tag or commit SHA (if using self-hosted version)

Release `2.18.0`

## Environment

 - Operating system: Windows 10
-Browser: Google Chrome

## Additional context

PFA screenshots.
![image](https://user-images.githubusercontent.com/102024574/185333915-3310d983-6783-44f7-8de8-b3211e583d09.png)

**Comment by ghost** [comment]
@AkshadaPramodKadam, thank you for opening this issue. We will triage it within the next few business days.

**Comment by azaslonov** [comment]
@AkshadaPramodKadam, since this is self-hosted portal, there is no backend to handle /signin-sso route properly (this is know problem and we're working on it). So, you can either create such page or have another existing on different route, but for this you'll need to change it [here](https://github.com/Azure/api-management-developer-portal/blob/master/src/components/defaultAuthenticator.ts#L28).

Hope that helps.

**Comment by ghost** [comment]
@AkshadaPramodKadam, we need more information before we start working on this issue. If you prefer to share it in private, please send us an email to apimportalfeedback@microsoft.com with the issue number in its subject.
